source/framework/vx_exemplar.c

```diff
@@ -252,6 +255,18 @@ static vx_reference ownCreatePyramidFromExemplar(
     pmd = vxCreatePyramid(context, levels, scale, width, height,
             format);
 
+    if ((vx_status)VX_SUCCESS == vxGetStatus((vx_reference)pmd))
+    {
+        vx_uint32 i;
+        vx_status status = (vx_enum)VX_SUCCESS;
+        for (i = 0; (i < levels) && ((vx_status)VX_SUCCESS == status); ++i)
+        {
+            if (exemplar->img[i]->base.supplementary_data)
```

Comment:
Can you please update the check to be the below for MISRA-C purposes?

if (NULL != exemplar->img[i]->base.supplementary_data)